api/room/schema_query.py

```diff
@@ -211,3 +232,35 @@ def resolve_bookings_analytics_count(self, info, start_date, end_date):  # noqa:
         analytics = RoomAnalyticsRatios.get_bookings_analytics_count(
             self, query, start_date, end_date)
         return analytics
+
+    def resolve_analytics_for_daily_room_events(self, info,
+                                                start_date, end_date
+                                                ):
+        start_date, end_date = CommonAnalytics().convert_dates(
+            start_date, end_date
+        )
+        query = Room.get_query(info)
+        all_events, all_dates = RoomSchedules().get_all_room_schedules(
+            query, start_date, end_date
+        )
+        all_days_events = []
```

Comment:
How is all_days_events different from this_days_events?

Reply:
All_days_events is events from all the days within the passed in range while this_days_events is the events for a particular day.